CONTRIBUTING.md

```diff
@@ -8,6 +8,8 @@ Hi! Thanks for checking out Julia. If you have questions or problems, the [Julia
 
 * [Fork Julia](https://github.com/JuliaLang/julia/fork).
 
+* Sign up for [Travis](https://travis-ci.org/profile/) and switch on your julia fork.
```

Comment:
Nah it can be left out, but it would be good to change the below line to say `./usr/bin/julia` instead of `./julia` so it works cross-platform.